api/api-cache.php

```diff
@@ -0,0 +1,132 @@
+<?php
+// these functions use $db, so scripts referencing them need to define it
+
+/**
+ * Gets the first entry from the API cache that has a matching endpoint.
+ * @param string $endpoint The endpoint for the API request
+ * @return array|null The first entry that matches or null if no entry exists
+ */
+function getFromApiCacheTable(string $endpoint)
```

Comment:
Don't we use snake case everywhere around the site?

Reply:
We should probably just name this `getCachedApiResponse` to make it clearer what we're returning

Reply:
Changed in 7cb4d48 and 15b78fa